shared/sszutil/deep_equal_test.go

```diff
@@ -59,7 +59,7 @@ func TestDeepEqualStructs_Unexported(t *testing.T) {
 
 func TestDeepEqualProto(t *testing.T) {
 	var fork1, fork2 pb.Fork
-	assert.Equal(t, true, sszutil.DeepEqual(fork1, fork2))
+	assert.Equal(t, true, sszutil.DeepEqual(&fork1, &fork2))
```

Comment:
Made these changes because I got 
```
Call of 'sszutil.DeepEqual' copies lock value: type 'pb.Fork' contains 'protoimpl.MessageState' contains 'sync.Mutex' which is 'sync.Locker' 
```
without the &

Reply:
Bump